frontera/contrib/backends/remote/messagebus.py

```diff
@@ -41,41 +45,41 @@ def frontier_stop(self):
         self.spider_log_producer.flush()
 
     def add_seeds(self, seeds):
-        per_host = aggregate_per_host(seeds)
-        for host_fprint, host_links in six.iteritems(per_host):
-            self.spider_log_producer.send(host_fprint,
-                                          self._encoder.encode_add_seeds(host_links))
+        key = b'0123456789abcdef0123456789abcdef012345678'
```

Comment:
There is no response nor request to calculate fprint from. The idea was not to perform aggregation at this point.

In overall, I dislike `b'0123456789abcdef0123456789abcdef012345678'` thing too, same goes for sending offsets after get_next_requests, since user might want to use different fprint calculation function. Most likely, I miss something important about how our message bus works, would be great if you could provide your opinion on what should be done here.